.github/workflows/e2e-tests.yml

```
@@ -184,7 +184,8 @@ jobs:
           --folder ${{ matrix.folder }}
         env:
           TERM: xterm
-          DEBUG: replay:cypress:plugin:events,replay:test-utils:reporter
+          DEBUG: replay:cypress:plugin:events,replay:test-utils:reporter,cypress:launcher*
+          RECORD_REPLAY_VERBOSE: 1
```

I've discovered that DeploySentinel does not pass on environment variables set by our plugin on to the browser. I'll follow up with them on that but, in the mean time, you can set the following env variable to get started.

```suggestion
          RECORD_REPLAY_METADATA_FILE: /tmp/replay-metadata.json
```